### PR TITLE
[14.0][MIG] pos_customer_display

### PR DESCRIPTION
**Pull request by alexis-via**
This is still a work in progress ; there's still some work to do on the JS. Help appreciated (I'm not a JS guy !).
I'll apply pre-commit scripts once the work on the JS code is finished.

### CODE REVIEW

File: pos_customer_display/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+This module adds support for LED Customer Display in the Point of Sale.
+
+It has been tested with the following devices:
+
+* Bixolon BCD-2000 (https://www.bixolon.com/product_view.php?idx=162),
```
Comment (legalsylvain) [inline pos_customer_display/readme/DESCRIPTION.rst:5]: Hi. did you tested with that device ? if yes, you could remove the roadmap part.

Comment (alexis-via) [reply pos_customer_display/readme/DESCRIPTION.rst:5]: I did test with BCD-2000 (I have one with me at the moment, cf the picture I sent on twitter), but it was configured in BCD-2000K mode with 2 x 20. So the roadmap entry is still valid; we could still improve the code to take advantage of 4x30. But I don't plan to implement that myself. For me, the priority is to improve/cleanup the current JS code, with the help of some JS devs.

**Comment by alexis-via** [comment]
FYI, this PR is used in production on odoo v14 since february 2021. The only thing lacking is fixing the JS warnings shown by pre-commit. I'm not very good in JS and I don't understand some of the warnings. So, if someone want to continue my work and fix the JS pre-commit warnings, feel free to superseed this PR.

**Comment by legalsylvain** [comment]
fixing pre-commit errors here : https://github.com/OCA/pos/pull/756

**Pull request closed without merging**